README.md

```diff
@@ -556,10 +556,10 @@ Create, reset, refresh, delete OAuth user-to-server tokens
         <code>repositoryIds</code>
       </th>
       <th>
-        <code>array of string</code>
+        <code>array of numbers</code>
       </th>
       <td>
-        The `id`s of the repositories that the installation token can access.
+        The <code>id</code> of the repositories that the installation token can access. Also known as a <code>databaseID</code> when querying the repository object in GitHub's v4 API.
```

Comment:
The v3/v4 terminologies are deprecated, we refer to them as REST API and GraphQL API now.

```suggestion
        The <code>id</code> of the repositories that the installation token can access. Also known as a <code>databaseID</code> when querying the repository object in GitHub's GraphQL API.
```

Reply:
Good to know 🎧 ❤️